server_handlers.go

```diff
@@ -70,6 +70,15 @@ func serverHandshakeHandler(c *Conn) error {
 				return err
 			}
 
+		case *handshakeMessageCertificateVerify:
+			if c.remoteCertificate == nil {
+				return errCertificateVerifyNoCertificate
+			}
+			if err := verifyCertificateVerify(c.handshakeCache.combinedHandshake(serverExcludeRules(), true), h.hashAlgorithm, h.signature, c.remoteCertificate); err != nil {
```

Comment:
Can you break long lines for readability? There are too many nested layers.

```
if err := verifyCertificateVerify(c.handshakeCache.combinedHandshake(serverExcludeRules(), true), h.hashAlgorithm, h.signature, c.remoteCertificate); err != nil {
```

vs

```
combined := c.handshakeCache.combinedHandshake(serverExcludeRules(), true)
err := verifyCertificateVerify(combined, h.hashAlgorithm, h.signature, c.remoteCertificate)
if err != nil {
```